compiler/src/dmd/frontend.h

```diff
@@ -7805,16 +7976,91 @@ struct Target final
 
 extern Target target;
 
-extern bool tpsemantic(TemplateParameter* tp, Scope* sc, Array<TemplateParameter* >* parameters);
+namespace dmd
+{
+    extern bool tpsemantic(TemplateParameter* tp, Scope* sc, Array<TemplateParameter* >* parameters);
 
-extern bool genTypeInfo(Expression* e, const Loc& loc, Type* torig, Scope* sc);
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
+namespace dmd
+{
+}
```

Comment:
I have no idea what's going on here with dtoh. It's harmless, but likely warrants a bug report.